lib/ansiblelint/__init__.py

```diff
@@ -22,123 +22,15 @@
 from collections import defaultdict
 import logging
 import os
-import re
 
+from ansiblelint.rules import AnsibleLintRule # noqa F401 backwards compatibility
```

Comment:
I'm pretty sure that the correct syntax would be
```suggestion
from ansiblelint.rules import AnsibleLintRule  # noqa: F401 
 # exposing the public API
```